_posts/en/newsletters/2021-01-06-newsletter.md

```diff
@@ -0,0 +1,177 @@
+  outputs][topic anchor outputs] the default for commitment transactions
+  and adds support for them in its [watchtower][topic watchtowers]
+  implementation, reducing costs and increasing safety.  The release also adds generic
+  support for creating and signing [PSBTs][topic psbt] and includes
+  several bug fixes.
+
+{% comment %}<!--
+- Bitcoin Core 0.20.2rc1 and 0.19.2rc1 are expected to be
+  [available][bitcoincore.org/bin] sometime after the publication of
+  this newsletter.  They contain several bug fixes, such as an
+  improvement described in [Newsletter #110][news110 bcc19620] that will
+  prevent them from redownloading future taproot transactions that they
+  don't understand.
+-->{% endcomment %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19137][] adds `dump` and `createfromdump` commands to [`wallet-tool`][news45
+  wallet-tool], allowing users to write wallet records to a dump file and subsequently
+  restore a wallet from that dump file. These commands are useful not only
+  for testing but as part of the [migration away from legacy wallets][achow101 legacy timeline].
+
+- [Bitcoin Core #20365][] updates the `bitcoin-wallet` tool's `create`
+  command with a `-descriptors` flag for creating [sqlite-backed
+  descriptor wallets][news120 sqlite], similar to the daemon's
+  [createwallet RPC][news96 createwallet].
+
+- [Bitcoin Core #20599][] updates the message handling code to process
+  `sendheaders` and `sendcmpct` messages received before the peer has sent its
+  `verack` message. The BIPs for `sendheaders` ([BIP130][]) and `sendcmpct`
+  ([BIP152][]) do not specify that those messages must be sent after the `verack`
+  message, and the original implementations ([Bitcoin Core #7129][] and
+  [Bitcoin Core #8068][]) allowed the messages to be received and processed
+  before the `verack` message. A later PR ([Bitcoin Core #9720][]) prevented
+  Bitcoin Core from processing those messages if they were received before the
+  `verack`. This PR restores the original behavior.
+
+- [Bitcoin Core #18772][] updates the `getblock` RPC when used with the
+  `verbosity=2` parameter to return a new `fee` field containing the
+  total fee for each transaction in the block.  This depends on
+  additional block undo data that nodes store in order to handle block
+  chain reorganizations.  Undo data is stored separately from other
+  block transaction data and nodes with pruning enabled may delete undo
+  data before deleting other data, so it's possible for pruned nodes to
+  sometimes return results without a `fee` field.
```

Comment:
nit: split infinitive
```suggestion
  data before deleting other data, so it's possible for pruned nodes
  sometimes to return results without a `fee` field.
```

Reply:
I don't think there's a problem with splitting infinitives. There's a good summary at https://en.wikipedia.org/wiki/Split_infinitive#Current_views.

For me, both this and the example below are clearer as they currently are - placing the modifier next to the verb that it's modifying.

Reply:
I don't think either suggestion diminishes the clarity, so I figured that these style suggestions were only a (marginal) net positive. However, this Wikipedia article has revealed that I'm an old-fashioned grammar luddite who needs to get with the times, so I'll stop flagging them.